lib/typescript/render/providers/facebook/FacebookRender.tsx

```diff
@@ -176,14 +202,14 @@ function facebookOutbound(message: Message): ContentUnion {
     };
   }
 
-  if (messageJson.attachment || messageJson.attachments) {
-    if ('text' in messageJson) {
-      return {
-        ...parseAttachment(messageJson.attachment || messageJson.attachments[0]),
-        text: messageJson.text,
-      };
-    }
+  if (messageJson.attachment?.type === 'fallback' || messageJson.attachments?.[0].type === 'fallback') {
+    return {
+      text: messageJson.text ?? null,
+      ...parseAttachment(messageJson.attachment || messageJson.attachments[0]),
+    };
+  }
 
+  if (messageJson.attachment || messageJson.attachments) {
     return parseAttachment(messageJson.attachment || messageJson.attachments[0]);
   }
 
```

Comment:
The message with fallback attachment can include a text (unlike other attachments)

- https://developers.facebook.com/docs/messenger-platform/reference/webhook-events/messages/